tools/Tools.Common/SerializedCmdlets/Az.Network.json

```diff
@@ -142052,7 +142052,24 @@
           },
           "ValidateSet": [
             "None",
-            "MaxConnections"
```

Comment:
Please revert the change of this file. It is generated by the tools

Reply:
Done